### DISCUSSION TITLE
Using pretrained wights for nnunetv2runner

### DISCUSSION THREAD

**Discussion by YazdanSalimi** [Project-MONAI/tutorials#1669] [Q&A]
I want to use my previously trained nnunet model and do transfer learning on a new dataset. the option offererd by the original reposirory (nnUNetv2_move_plans_between_datasets ) needs preprocessed data. I dis not keep the preprocessed data, is there a way to preprocess the new dataset in the same manner as the old one with -pretrained_weights option in train_single_model function?
Thank you

**Comment by KumoLiu** [comment]
Hi @YazdanSalimi, for preprocessing new datasets, it would be ideal to make a reference to the nnUNet repository as train_single_model does not inherently handle that. However, it could be beneficial for `nnUNetV2Runner` to encompass an enhancement to support this feature in an upcoming update. This potential enhancement could streamline our workflow.
https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/pretraining_and_finetuning.md

Thanks.
